Description: Correct pod NAME issue in LedgerSMB/Database.pm
 Has the module name but not the dash plus short description in the NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472579&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/Database.pm
+++ b/LedgerSMB/Database.pm
@@ -1,7 +1,7 @@
 #!/usr/bin/perl
 =head1 NAME
 
-LedgerSMB::Database
+LedgerSMB::Database - Provides the APIs for database creation and management
 
 =head1 SYNOPSIS
 
